sanket/javascript/assignment12(day25)/sanketA12_Q1.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>employee details</title>
+  </head>
+  <body>
+    <script>
+      class Employee {
+        constructor() {
+          this.empNo = 101;
+          this.name = "Ashok Kumar";
+          this.address = "43 Ramnagar";
+          this.city = "Pune Maharashtra";
+          this.salary = 80000;
```

Comment:
**create da, hra, pf class variable**
for this.gross empty class variable
**like: this.da, etc.**

Reply:
if you are using variables like this.da is instance variable not class variable, also i am not going to use these variable outside the function, so creation an instance variable is a vary costly operation if you are not going to use it any where in the program do not create instance variable

Reply:
Ok